Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.cs

```diff
@@ -200,6 +183,8 @@ public RadialGauge()
         {
             DefaultStyleKey = typeof(RadialGauge);
 
+            SmallChange = 1;
```

Comment:
Can we set them in the style instead of here?

I don't like changing the Large from 5 to 10 in a point release, but everything else looks good.

Reply:
Yeah, tested moving them to style works fine.  We can get this merged in and update quick tomorrow during our bug bash to not hold up this PR.  @nmetulev @azchohfi let me know if you can give me the 2nd approval quick.  Thanks!

Reply:
Agree on moving these to the style but that's also a breaking change if someone re-templated before this update. My vote is for including this in 6.0 and document the breaking change

Reply:
This should not be moved to the style. We are making a new default for the property. The only way to do this is through code.